README.md

```diff
@@ -11,21 +11,17 @@ from v1.0.  Working branch v1.0a, becomes v2.0 on release
 6.  Run `poetry install`
 
 ## Running `rctGCS`
--   Windows:
-    1.  Run `rctGCS.bat` by double clicking or running from command line
--   Ubuntu/MacOS:
-    1.  From the command line, run `conda activate rctGCS`
-    2.  Navigate to the `scripts` folder.
-    3.  Run `python rctGCS.py`
+1.  From the command line, run `conda activate rctGCS`
+2.  Run `RCTGcs`
     
 ## Running the simulator for GCS
 1. Navigate to the `scripts` folder and run `rctGCS` with the following commands:
-    - `$ conda activate rctGCS`
-    - `python rctGCS.py`
+    - `conda activate rctGCS`
+    - `RCTGcs`
     This should open up the rctGCS UI
 2. Create a new terminal instance and naviagate to the `scripts` folder and run the following commands:
```

Comment:
We don't have a `scripts` folder anymore, so lines 18 and 22 should specify navigation to `RctGcs`. Also, I believe lines 15 and 20 should be `rctGCS` to match capitalization of the Python file.